src/editors/containers/VideoEditor/components/VideoSettingsModal/components/LicenseWidget/LicenseDisplay.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  FormattedMessage,
+  injectIntl,
+} from '@edx/frontend-platform/i18n';
+import {
+  Card,
+  Form,
+  Hyperlink,
+} from '@edx/paragon';
+
+import { LicenseLevel, LicenseTypes } from '../../../../../../data/constants/licenses';
+
+import LicenseBlurb from './LicenseBlurb';
+import { messages } from './messages';
+/**
+ * Collapsible Form widget controlling videe license type and details
+ */
+// TODO change placeholder icons for NC, ND, SA
+export const LicenseDisplay = ({
+  license,
+  details,
+  licenseDescription,
+  level,
+}) => (
+  license !== LicenseTypes.select ? (
+    <div className="border-primary-100 border-top pb-4">
+      <Form.Group>
```

Comment:
Why is this a form group? For visual purposes? Could we not make the form group at the same component level as the form control?